File: frameworks/go/fiber/Implementation/json.go
package implementation

import (
	"github.com/gofiber/fiber/v3"
)

// jsonRoutes serialise a payload the framework already holds, at three sizes. c.JSON encodes
// it with the app's JSONEncoder, encoding/json's Marshal by default, and sets it as the body.
func jsonRoutes(app *fiber.App, p *Payloads) {
	app.Get("/json/small", func(c fiber.Ctx) error { return c.JSON(&p.Small) })

	app.Get("/json/medium", func(c fiber.Ctx) error { return c.JSON(&p.Medium) })

	app.Get("/json/large", func(c fiber.Ctx) error { return c.JSON(&p.Large) })
}

File: frameworks/rust/actix-web/Implementation/routes/json.rs
use actix_web::web::{self, ServiceConfig};

use crate::Payloads;

/// json: a payload the framework already holds, serialised at three sizes by actix-web's Json
/// responder.
pub fn configure(cfg: &mut ServiceConfig, p: &'static Payloads) {
    cfg.route("/json/small", web::get().to(move || async move { web::Json(&p.small) }))
        .route("/json/medium", web::get().to(move || async move { web::Json(&p.medium) }))
        .route("/json/large", web::get().to(move || async move { web::Json(&p.large) }));
}
